### DISCUSSION TITLE
Broken?

### DISCUSSION THREAD

**Discussion by HavenKotara** [janbaykara/ebook-scraper#50] [General]
Spent a few hours trying to debug and get the extension to work.
I'm attempting to use it for ProQuest and the extension isn't working properly. 

In Chrome://extensions these errors are shown: 
`Unchecked runtime.lastError: The message port closed before a response was received.` (popup.html)
'Uncaught (in promise) Error' (js/eventPage.js:377)

And when inspecting the extension the console shows:
```
fetching book (Popup.tsx:38)
Received book > Object (Popup.tsx:57)
null (index.ts:77)
null (index.ts:77)
```

The extension seems to have trouble getting the pages.

**Comment by danitt** [comment]
try swapping to PDF view as described here
https://github.com/janbaykara/ebook-scraper/issues/37#issuecomment-3118495096

**Comment by danitt** [comment]
closing stale discussion
